requirements-cuda.txt

```diff
@@ -5,5 +5,6 @@
 ray >= 2.9
 nvidia-ml-py # for pynvml package
 torch == 2.3.0
+torchvision == 0.18.0   # required for the image processor of phi3v
```

Comment:
Can you add a link to https://github.com/pytorch/vision?tab=readme-ov-file#installation and note that this must be updated alongside torch